src/app.js

```diff
@@ -6,8 +6,220 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
+// Our components
+import Trip from './app/models/trip';
+import Reservation from './app/models/reservation';
+import TripList from './app/collections/trip_list';
 console.log('it loaded!');
 
-$(document).ready( () => {
-  $('main').html('<h1>Hello World!</h1>');
+const tripList = new TripList()
+
+
+const TRIP_FIELDS = ['name', 'continent', 'about', 'category', 'weeks', 'cost'];
+const RESERVATION_FIELDS = ['name', 'email', 'age'];
+
+let tripsTemplate;
+let tripTemplate;
+
+// Clear status messages
+const clearStatus = function clearStatus() {
+  $('#status-messages ul').html('');
+  $('#status-messages').hide();
+};
+
+// Add a new status message
+const reportStatus = function reportStatus(status, message) {
+  console.log(`Reporting ${ status } status: ${ message }`);
+
+  const statusHTML = `<li class="${ status }">${ message }</li>`;
+
+  // note the symetry with clearStatus()
+  $('#status-messages ul').append(statusHTML);
+  $('#status-messages').show();
+};
+
+const render = function render(tripList){
+  const $tripsTableElement = $('#trip-list');
+  $tripsTableElement.html('');
+
+  tripList.forEach((trip) => {
+    const generatedHTML = $(tripsTemplate(trip.attributes));
+    generatedHTML.on('click',(event) => {
+      $('#trips').hide();
+      $('#trip').show();
+      trip.fetch({
```

Comment:
Consider adding an `error` event handler to deal with times that the API doesn't work and let the user know.